src/app/app.module.ts

```diff
@@ -3,6 +3,7 @@ import { NgModule, ErrorHandler } from '@angular/core';
 import { HttpModule } from '@angular/http';
 import { IonicApp, IonicModule, IonicErrorHandler } from 'ionic-angular';
 import { IonicStorageModule } from '@ionic/storage';
+import { BrowserModule } from '@angular/platform-browser';
```

Comment:
BrowserModule (re)exports stuff like `ngIf` for use in browser apps, but also DOM stuff or Web worker. Before this stuff was provided by another import, now it was packaged together here.